traffic_ops/testing/api_contract/v4/test_servers.py

```diff
@@ -41,8 +40,8 @@ def test_server_contract(to_session: TOSession,
 	"""
 	# validate server keys from server get response
 	logger.info("Accessing /servers endpoint through Traffic ops session.")
-	profile_id = server_post_data[1]
-	server_post_data = server_post_data[0]
+	# profile_id = server_post_data[1]
```

Comment:
shell we keep this commented lines of code

Reply:
Yes, We can remove that.